src/actions.ts

```diff
@@ -0,0 +1,10 @@
+import { UserAction } from './common/services/user';
+import { SpinnerActions } from './packages/spinner/ducks';
+
+type RootAction = UserAction | SpinnerActions;
```

Comment:
arent there also some router actions?

Reply:
netreba

Reply:
preco netreba? naco nam vlastne je ten `RootAction ` ?

Reply:
je to podla typesafe-actions docs

Reply:
aha ok, checknem si to poriadne

Reply:
len vidim ze to nikde nepouzivame, tak ma zaujimalo kde to chceme pouzit